pep-0373.txt

```diff
@@ -91,6 +90,7 @@ Dates of previous maintenance releases:
 - 2.7.16 2019-03-02
 - 2.7.17rc1 2019-10-05
 - 2.7.17 2019-10-19
+- 2.7.18 release candidate 2020-04-04
```

Comment:
For consistency with others in this list:
```suggestion
- 2.7.18rc1 2020-04-04
```
Re: https://www.python.org/downloads/release/python-2718rc1/

Reply:
Thanks, good idea! Fixed.